guides/small_prefixes.md

```diff
@@ -50,3 +50,24 @@ prefix set too_small; {
     return true;
 }
 ```
+
+## Cisco classic IOS and IOS XE
+```
+ip prefix-list peerfilter seq 5 deny 0.0.0.0/0
+ip prefix-list peerfilter seq 10 permit 0.0.0.0/0 ge 8 le 28
```

Comment:
All the other examples use le 27 or le28; and RIPE has given out several PI blocks size 25-27. So does the text above the examples.